### PR TITLE
Fixed page [close #14]

### LINKED ISSUE #14: 利用規約やプライバシーポリシーをどうする？
サービス固有になると思われるので、その設定(切り替え?)方法をどうしよう…

### PR DESCRIPTION
**Pull request by mtsmfm**

### CODE REVIEW

File: db/migrate/20151210063912_create_fixed_pages.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CreateFixedPages < ActiveRecord::Migration
+  def change
+    create_table :fixed_pages do |t|
+      t.references :site, foreign_key: true, null: false
+
+      t.string :title
+      t.text   :body
+      t.string :slug
```
Comment (tricknotes) [inline db/migrate/20151210063912_create_fixed_pages.rb]: Needs `null: false` ?

Comment (mtsmfm) [inline db/migrate/20151210063912_create_fixed_pages.rb]: good catch :baseball:

**Comment by myokoym** [comment]
I meet the following error...

```
% bin/rake db:fixtures:load
rake aborted!
ActiveRecord::InvalidForeignKey: PG::ForeignKeyViolation: ERROR:  テーブル"fixed_pages"への挿入、更新は外
部キー制約"fk_rails_400f45177e"に違反しています
DETAIL:  テーブル"sites"にキー(site_id)=(617777887)がありません
: INSERT INTO "fixed_pages" ("slug", "title", "body", "created_at", "updated_at", "id", "site_id") VALUES
 ('about', 'About Site 1', '# About Site 1

Site 1 is:

- A
- B
- C
```

**Comment by mtsmfm** [comment]
Try following command:

`$ bin/rake db:migrate:reset db:fixtures:load`

**Comment by mtsmfm** [comment]
If you met same error, try this one:

`bin/rake db:fixtures:load FIXTURES=sites,categories,posts,fixed_pages`

**Pull request merged**